apps/landingPage/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "gatsby-starter-default",
```

Comment:
maybe we should name this package differently?

Reply:
Just changed with commit